test/core/end2end/tests/call_creds.cc

```diff
@@ -264,11 +264,17 @@ CORE_END2END_TEST(PerCallCredsOnInsecureTest,
 }
 
 CORE_END2END_TEST(PerCallCredsTest, RequestResponseWithPayloadAndCallCreds) {
+  if (IsLocalConnectorSecureEnabled()) {
+    SKIP_IF_LOCAL_TCP_CREDS();
```

Comment:
Why do we need a special case here for local TCP credentials when we don't seem to have one for insecure creds?  Presumably these tests also don't work for insecure creds, but we don't have a special case here to exclude insecure creds.  That implies to me that there's some other mechanism to exclude these tests for insecure creds, and I'm wondering why we can't use the same mechanism for both.

Reply:
We discussed offline with @ctiller that it's ok to leave it as is for now (Craig, pls correct me if I'm wrong). Once experiment is done, we can reorganize the suites.